caikit/runtime/service_generation/data_stream_source.py

```diff
@@ -116,14 +116,14 @@ def to_data_stream(self) -> DataStream:
             return DataStream.from_iterable(self.jsondata.data)
 
         # If jsonfile, attempt to read the file and pull in the data from there
-        if set_field == "file":
-            return self._create_data_stream_from_file(fname=self.file.filename)
+        if set_field == "filestream":
```

Comment:
While we're in this class, do you think it would be possible to implement at least `from_file` so that for data stream parameters, users can send in like:
```
inputs.my_data_stream_arg: raw_data
``

Reply:
@joerunde I'm not quite sure what you mean. Like update `_create_data_stream_from_file` to support both filename and raw bytes?